tests/baselines/reference/ambientDeclarations.js

```diff
@@ -77,7 +77,19 @@ declare module 'external1' {
 
 
 //// [ambientDeclarations.js]
+// Ambient variable with type annotation
+// Ambient function with no type annotations
+// Ambient function with type annotations
+// Ambient function with valid overloads
+// Ambient function with optional parameters
+// Ambient class
+// Ambient enum
+// Ambient enum with integer literal initializer
+// Ambient enum members are always exported with or without export keyword

```

Comment:
> Can we define what are precisely the conditions to hide or show comments?

Honestly, that's the hard part (if we had a good rule, we would have done it already). Clearly leading comments for type declarations shouldn't be present in the emit as they look like misleading nonsense, but comments between an opening brace of a statement block and another expression statement should be present.

Figuring out where it makes sense to emit the comments and where it doesn't is the hard part. Simply emitting *every* comment isn't the right fix.